ruby/ql/lib/codeql/ruby/frameworks/http_clients/Faraday.qll

```diff
@@ -37,7 +37,8 @@ class FaradayHttpRequest extends Http::Client::Request::Range, DataFlow::CallNod
         API::getTopLevelMember("Faraday").getInstance()
       ] and
     requestNode =
-      connectionNode.getReturn(["get", "head", "delete", "post", "put", "patch", "trace"]) and
+      connectionNode
+          .getReturn(["get", "head", "delete", "post", "put", "patch", "trace", "run_request"]) and
```

Comment:
Perhaps this change belongs to another PR?

Reply:
I could have made a separate PR for it indeed. It was such a minor change that I left it in .